scripts/gen_node

```diff
@@ -1,62 +0,0 @@
-#! /usr/bin/env python3
-
-import argparse
-
-from sys import argv
-
-from plenum.common.script_helper import getStewardKeyFromName, \
-    generateNodeGenesisTxn, buildKeepDirIfNotExists
-from plenum.common.constants import CLIENT_STACK_SUFFIX
-from plenum.common.keygen_utils import initNodeKeysForBothStacks
-from stp_core.crypto.util import isHexKey
-from plenum.common.config_util import getConfig
-
-config = getConfig()
-
-if __name__ == "__main__":
```

Comment:
Why did we remove this script? How is it related to RAET?

Reply:
Script returned